config/development.json

```diff
@@ -1,4 +1,4 @@
 {
   "env": "development",
-  "publicPath": "/"
+  "publicPath": "/v2"
```

Comment:
I don't know if this will cause any problems, but you can leave the pulicPath as it was before.